src/bridge/cockpitpackages.c

```diff
@@ -884,7 +884,7 @@ package_content (CockpitPackages *packages,
       g_free (chosen);
       chosen = NULL;
 
-      bytes = cockpit_web_response_negotiation (filename, package->paths, language, &chosen, &error);
+      bytes = cockpit_web_response_negotiation (filename, package ? package->paths : NULL, language, &chosen, &error);
```

Comment:
> then cockpit_packages_resolve() would be broken

it doesn't touch `package` at all if some of the input variables are `NULL`: https://github.com/dperpeet/cockpit/blob/81949a275817fb380d01799336e870d5b9b5d37f/src/bridge/cockpitpackages.c#L719